src/Behat/Symfony2Extension/ServiceContainer/Symfony2Extension.php

```diff
@@ -72,8 +72,8 @@ public function configure(ArrayNodeDefinition $builder)
                     ->addDefaultsIfNotSet()
                     ->children()
                         ->scalarNode('bootstrap')->defaultValue(self::DEFAULT_KERNEL_BOOTSTRAP)->end()
```

Comment:
this must be updated as well, as Symfony 4 does not have this boostrap file anymore either